Stripe/STPAPIRequest.m

```diff
@@ -9,120 +9,183 @@
 #import "STPAPIRequest.h"
 
 #import "NSMutableURLRequest+Stripe.h"
-#import "STPInternalAPIResponseDecodable.h"
-#import "STPAPIClient+Private.h"
 #import "STPAPIClient.h"
-#import "STPCard+Private.h"
+#import "STPAPIClient+Private.h"
 #import "STPDispatchFunctions.h"
-#import "STPFormEncoder.h"
-#import "STPSource+Private.h"
+#import "STPInternalAPIResponseDecodable.h"
 #import "StripeError.h"
 
 @implementation STPAPIRequest
 
+static NSString * const HTTPMethodPOST = @"POST";
+static NSString * const HTTPMethodGET = @"GET";
+static NSString * const HTTPMethodDELETE = @"DELETE";
+
+static NSString * const JSONKeyObject = @"object";
```

Comment:
This seems good to me! And yeah, I don't think we have to worry about namespacing these